src/hooks/useSuggestions.ts

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useState } from 'react'
+
+import { SickObj, suggestionAPI } from '../apis/suggestion'
+
+const useSuggestions = (debouncedValue: string) => {
```

Comment:
useSuggestions훅의 입장에서는, 주입받는 의존성 (debouncedValue)이 debounced된 값인지 까지는 알 필요 없다고 생각해서 네이밍 정도만 다음과 같이 바꾸면 좋을것 같습니다!

```suggestion
const useSuggestions = (query: string) => {
```
```suggestion
const useSuggestions = (keyword: string) => {
```

Reply:
아하 !
그러면 query보다는 keyword가 친숙해서 ,,, keyword로 바꾸도록 하겠습니다 !!